gulpfile.js

```diff
@@ -1,261 +1,18 @@
-var fs = require('fs');
-// Make sure we have a Config.dev so we don't error on Config loading
-var configFilePath = './src/js/config/Config.dev.js';
-if (!fs.existsSync(configFilePath)) {
-  fs.writeFileSync(configFilePath, 'module.exports = {};', 'utf8');
-}
-// dependencies
-var autoprefixer = require('gulp-autoprefixer');
-var browserSync = require('browser-sync');
-var colorLighten = require('less-color-lighten');
-var changed = require('gulp-changed');
 var connect = require('gulp-connect');
-var del = require('del');
+var fs = require('fs');
 var gulp = require('gulp');
-var gulpif = require('gulp-if');
-var gutil = require('gulp-util');
-var imagemin = require('gulp-imagemin');
-var less = require('gulp-less');
-var csso = require('gulp-csso');
-var mkdirp = require('mkdirp');
-var path = require('path');
-var replace = require('gulp-replace');
-var runSequence = require('run-sequence');
-var sourcemaps = require('gulp-sourcemaps');
-var uglify = require('gulp-uglify');
-var webpack = require('webpack');
-var WebpackNotifierPlugin = require('webpack-notifier');
-
-var config = require('./.build.config');
-var packageInfo = require('./package');
-var webpackConfig = require('./.webpack.config');
-
-var development = process.env.NODE_ENV === 'development';
-var devBuild = development || (process.env.NODE_ENV === 'testing');
-var externalPluginsDirectory = '../dcos-ui-plugins-private';
-
-var pluginsGlob = [
-  externalPluginsDirectory + '/**/*.*'
-];
-
-function browserSyncReload() {
-  if (development) {
-    browserSync.reload();
-  }
-}
-// Watches for delete in external plugins directory and deletes counterpart in
-// DC/OS-UI directory
-function deletePluginFile(event) {
-  if (event.type === 'deleted') {
-    var filePathFromPlugins = path.relative(
-      path.resolve(externalPluginsDirectory), event.path);
-
-    var destFilePath = path.resolve(
-      config.dirs.pluginsTmp, filePathFromPlugins);
-
-    del.sync(destFilePath);
-    webpackFn(browserSyncReload);
-  }
-}
 
-function ensurePluginDirectoryExists() {
-  // Make temp plugins directory if doesn't exist
-  if (!fs.existsSync(config.dirs.pluginsTmp)) {
-    mkdirp(config.dirs.pluginsTmp, function () {
-      console.log('Created ' + config.dirs.pluginsTmp);
-    });
+gulp.task('ensureConfig', function () {
```

Comment:
Thats fair, but I think since we have it, it doesn't make sense to remove it since its helping people get up and running quicker. It's nice for someone to clone the repo and `npm i && npm start`. In exchange for removing this, we'd have to write more docs (which we need at some point, just not immediately) :). What do you think?